groups/sensors.yml

```diff
@@ -0,0 +1 @@
+name: Sensors
```

Comment:
Can you add a comment to say if the intended scope of the group is only the APIs built on https://w3c.github.io/sensors/, or also older features that are also exposing sensor data, like device motion events?

Reply:
No strong opinion on this. I would tend to make this group pretty generic and accommodate all types of features that expose sensor data. We could then perhaps create subgroups for new generic sensor API-based things, and older parts.

Reply:
OK, let's put `device-orientation-events` into this group too then.